src/commons/jira/client.py

```
@@ -0,0 +1,420 @@
+    def get_available_transitions(self, issue: Issue) -> List[Dict[str, str]]:
+        """Get available transitions for an issue
+
+        Args:
+            issue: JIRA issue
+
+        Returns:
+            List of transition dictionaries with 'id' and 'name' keys
+        """
+        try:
+            return self._retry_operation(self.jira.transitions, issue)
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)